src/patternfly/upgrade-examples/LoginUpgradeExamples/examples/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import Documentation from '@siteComponents/Documentation';
+import Example from '@siteComponents/Example';
+import docs from '../docs/code.md';
+import loginUpgradeExamplePF3 from './login-upgrade-examples-pf3-login.hbs';
+import loginUpgradeExamplePF4 from './login-upgrade-examples-pf4-login.hbs';
```

Comment:
Since you're just duplicating the pf4 login component (is that right?), I wonder if we should just point directly to that component hbs file instead of having a copy. If we end up changing the login component, and want it reflected in the upgrade example, we have 2 files to edit now, not 1.

Reply:
That's one way to save on file load, space. Has anyone done anywhere else in the system?

Reply:
I don't think so. Not sure if that's a good idea or not. If this login demo should match what's in the PF4 component, I think it's a good idea. If we want to customize this demo in a way that deviates from the PF4 component, probably best for it to be it's own code here. I don't think it really matters either way :)

Reply:
As of right now, I think keeping the code separate (even if they are the same) is designed for the comparison portion for the future PF site.

Reply:
I agree with @mcoker on this one - the example hbs will be available from either her or the website as assets so separation is not a concern.

Reply:
Hey @mcoker / @dgutride, if we point to the PF4 login component files, this will bring back the background image and social media login icons. It was suggested me to remove the icons at one point, but with further examination, @mcoker suggested that it is best to keep the login upgrade example demo separate from the login component. 

Any final thoughts?